images/commons/entrypoint-readiness

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
```

Comment:
can we switch this to just `sh`? not all images have bash installed all the time.